### PR TITLE
Add subprocess KB article

### PR DESCRIPTION
**Pull request by snehankekre**
- Adds a KB article about [invoking subprocesses](https://www.notion.so/streamlit/Docs-Inbox-ee8e809c685a476dbb2667ecb46e5c51?p=4209deb63cf748b0859f9eea9921affb) to run Python scripts in deployed apps.

### CODE REVIEW

File: content/kb/deployments/invoking-python-subprocess-deployed-streamlit-app.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Invoking a Python subprocess in a deployed Streamlit app
+slug: /knowledge-base/deploy/invoking-python-subprocess-deployed-streamlit-app
+---
+
+# Invoking a Python subprocess in a deployed Streamlit app
+
+## Problem
+
+Let's suppose you want to invoke a subprocess to run a Python script `script.py` in your deployed Streamlit app `streamlit_app.py`.
```
Comment (randyzwitch) [inline content/kb/deployments/invoking-python-subprocess-deployed-streamlit-app.md:10]: As we discussed, maybe add another sentence like:

"For example, the popular machine learning library Ludwig is run using a command-line interface, or maybe you want to run a bash script or similar type of process..."

**Review by randyzwitch — APPROVED** [review]
Approved pending minor comment

File: content/kb/deployments/invoking-python-subprocess-deployed-streamlit-app.md (hunk 1 above)
Comment (randyzwitch) [reply content/kb/deployments/invoking-python-subprocess-deployed-streamlit-app.md]: Added it in, take a look @snehankekre just to make sure it seems reasonable, then this is ready for merge.

Comment (snehankekre) [reply content/kb/deployments/invoking-python-subprocess-deployed-streamlit-app.md]: Thanks for making the change, @randyzwitch. LGTM!

**Pull request merged**